extensions/pkg/util/shoot.go

```diff
@@ -42,3 +45,15 @@ func VersionInfo(vs string) (*version.Info, error) {
 		GitVersion: fmt.Sprintf("v%d.%d.%d", v.Major(), v.Minor(), v.Patch()),
 	}, nil
 }
+
+// IsPSPDisabled returns true if the PodSecurityPolicy plugin is explicitly disabled in the ShootSpec
+func IsPSPDisabled(shoot *gardencorev1beta1.Shoot) bool {
+	if shoot.Spec.Kubernetes.KubeAPIServer != nil {
+		for _, plugin := range shoot.Spec.Kubernetes.KubeAPIServer.AdmissionPlugins {
+			if plugin.Name == "PodSecurityPolicy" && pointer.BoolDeref(plugin.Disabled, false) {
```

Comment:
Should we use some constant for `PodSecurityPolicy `?

Reply:
Hmm, I vote against it.